### DISCUSSION TITLE
How to change path to wildcards?

### DISCUSSION THREAD

**Discussion by lgwjames** [lllyasviel/Fooocus#2443] [Q&A] [answered]
i want to take out some files on my main drive to free up space

how do i change my config file to point to wildcards on my ssd?

{
    "path_checkpoints": "/Volumes/Ai works/Checkpoints",
    "path_loras": "/Volumes/Ai works/Loras",
    "path_embeddings": "/Users/Quad-Core/Fooocus/models/embeddings",
    "path_vae_approx": "/Users/Quad-Core/Fooocus/models/vae_approx",
    "path_upscale_models": "/Users/Quad-Core/Fooocus/models/upscale_models",
    "path_inpaint": "/Users/Quad-Core/Fooocus/models/inpaint",
    "path_controlnet": "/Users/Quad-Core/Fooocus/models/controlnet",
    "path_clip_vision": "/Users/Quad-Core/Fooocus/models/clip_vision",
    "path_fooocus_expansion": "/Users/Quad-Core/Fooocus/models/prompt_expansion/fooocus_expansion",
    "path_outputs": "/Volumes/Ai works",
    "default_prompt_negative": "__ neg __",
    "default_refiner": "",
    "default_prompt": "",
    "default_loras_min_weight": -8,
    "default_loras_max_weight": 8,  
    "default_advanced_checkbox": true,
    "default_max_image_number": 12,
    
    "default_loras": [
        [
            "JessicaL.safetensors",
            1.0
        ],
        [
            "PerfectEyesXL.safetensors",
            1.0
        ],
        [
            "Perfect Hands v2.safetensors",
            1.0
        ],
        [
            "None",
            1.0
        ],
        [
            "None",
            1.0
        ]
],
       
    "default_image_number": 4

}

**Comment by ILikeToasters** [comment, marked as answer]
As far as I know that is not something you can change in the config or presets from looking over the code quickly. But will mention those files take up almost zero space.

**Comment by mashb1t** [comment]
This won't help in your quest to clean up your system. Please check which checkpoints and/or LoRAs you don't need anymore and delete them instead.
Wildcards are only text files, ~5-10kb max each.
